### PR TITLE
Configuration rebuild.

### PR DESCRIPTION
**Pull request by dannon**
Not exactly sure when this got out of sync looking at the immediate history, but this popped up in another context as needing a rebuild.

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [x] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

**Comment by martenson** [comment]
we should have a CI for this

**Review by martenson — APPROVED** [review]

File: lib/galaxy/config/sample/galaxy.yml.sample [hunk 1]
```diff
@@ -246,86 +161,20 @@ gravity:
     # upload_dir:
 
     # Comma-separated string of enabled tusd hooks.
-    #
+    # 
     # Leave at the default value to require authorization at upload creation time.
     # This means Galaxy's web process does not need to be running after creating the initial
     # upload request.
-    #
+    #
```
Comment (martenson) [inline lib/galaxy/config/sample/galaxy.yml.sample:168]: this is weird?

Comment (martenson) [reply lib/galaxy/config/sample/galaxy.yml.sample:168]: I guess standard

Comment (dannon) [reply lib/galaxy/config/sample/galaxy.yml.sample:168]: :shrug:  I just did `make config-rebuild`

**Comment by mvdbeek** [comment]
huh, this is verting nate's changes ... are you on a recent dev ?

**Comment by dannon** [comment]
Swapped to draft -- it looks like this was a lot in https://github.com/galaxyproject/galaxy/pull/15017

**Comment by dannon** [comment]
@mvdbeek Yeah, I just saw that too when I went to track down where it came from.   None of that is in the schema and gets overwritten on the latest dev for me -- how is this supposed to be handled?

**Comment by nsoranzo** [comment]
@dannon Did you update your .venv with the latest gravity?

**Comment by dannon** [comment]
@nsoranzo Ah-ha!  Let me try that.  I see in config_manage where it should be pulling that in now.

**Comment by dannon** [comment]
Fancy configuration merging there, I just learned something -- thanks @nsoranzo

**Pull request closed without merging**